packages/commcare/test/index.js

```diff
@@ -653,3 +653,37 @@ describe('createUser', () => {
     expect(response.statusCode).to.equal(201);
   });
 });
+
+
+describe('HTTP wrappers', () => {
+  it('makes a GET request', async () => {
+    testServer
+      .intercept({
+        path: `/a/asri/api/v0.5/case`,
+        method: 'GET',
+      })
+      .reply(200, () => {
+        // simulate a return from commcare
+        return {
+          "case_id": "1",
+        };
+      });
+
+    const state = {
+      configuration: {
+        hostUrl,
+        domain,
+        appId: app,
+        username: 'user',
+        password: 'password',
+      },
+    };
+
+    const { data, response } = await 
+      request('GET', '/a/asri/api/v0.5/case')
```

Comment:
It would be great to use a bit more of the API in this test. Do you think you could add a query or something just to make sure the params object gets fed through?

You'll need to update the interceptor to also match on the query pattern